fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingScreen.kt

```diff
@@ -65,14 +65,13 @@ fun OnboardingScreen(
     pagesToDisplay: List<OnboardingPageUiData>,
     onMakeFirefoxDefaultClick: () -> Unit,
     onSkipDefaultClick: () -> Unit,
-    onPrivacyPolicyClick: (url: String) -> Unit,
     onSignInButtonClick: () -> Unit,
     onSkipSignInClick: () -> Unit,
     onNotificationPermissionButtonClick: () -> Unit,
     onSkipNotificationClick: () -> Unit,
     onAddFirefoxWidgetClick: () -> Unit,
     onSkipFirefoxWidgetClick: () -> Unit,
-    onFinish: (pageType: OnboardingPageUiData) -> Unit,
+    onFinish: (pageType: OnboardingPageUiData?) -> Unit,
```

Comment:
This could remain  `onFinish: (pageType: OnboardingPageUiData) -> Unit` since it won't callback with null here

Reply:
Make sense, however, if we change it back we'll have to have two different `onFinish` functions. I think keeping it with the nullable signature is simpler.

Reply:
> If we change it back we'll have to have two different onFinish functions. 

🤔 I doubt that, can you please double check? This wouldn't affect the onFinish in `OnboardingFragment`

Reply:
Ah I see, we're not passing in `onFinish()` directly, but as a lambda that calls `onFinish`.  Sure I'll update.  Thanks